internal/armada/server/submit_from_log.go

```diff
@@ -146,7 +145,7 @@ func (srv *SubmitFromLog) Run(ctx context.Context) error {
 			messageLogger.WithField("numEvents", len(sequence.Events)).Info("processing sequence")
 			// TODO: Improve retry logic.
 			srv.ProcessSequence(ctxWithLogger, sequence)
-			srv.Consumer.Ack(msg)
+			srv.ack(ctx, msg)
```

Comment:
If the context expires before getting here, we don't even ack here at all right?

Reply:
Actually the context here never expires unless the serer is shutting down, so probably fine